docs/conf.py

```diff
@@ -236,7 +236,8 @@
     'mongodb': ('https://api.mongodb.com/python/current/', None),
     'pandas': ('https://pandas.pydata.org/pandas-docs/stable/', None),
     'python': ('https://docs.python.org/3/', None),
-    'requests': ('http://docs.python-requests.org/en/master/', None),
+    # Re-enable when python-requests.org is working again!
```

Comment:
```suggestion
    # TODO: Re-enable when python-requests.org is working again!
```
After that, you can merge it..